ui/app/com/gu/recipeasy/controllers/Application.scala

```diff
@@ -34,7 +34,7 @@ class Application(override val wsClient: WSClient, override val conf: Configurat
   def viewRecipe(id: String) = AuthAction { implicit request =>
     val recipe = db.getOriginalRecipe(id)
     db.insertUserEvent(UserEvent(request.user.email, request.user.firstName, request.user.lastName, id, UserEventAccessRecipeReadOnlyPage.name))
-    curatedRecipedEditor(recipe, editable = false)
+    curatedRecipedEditor(recipe, editable = false, userEmail = request.user.email, userFirstName = request.user.firstName, userLastName = request.user.lastName)
```

Comment:
Um.... I thought of that, but I decided to reproduce what I did for the main page. I know it's not as beautiful, but the type of the user itself is... funny.